### ISSUE TITLE
Unable to resolve .fun tld

### ISSUE THREAD

**Issue by nenkoszai** [eithan1231/whois-light#6]
This one is  specifically important as more criminals are starting to register .fun to spam their phishing domains on discord. 

I hope you can add a support for this asap.

**Comment by eithan1231** [comment]
Hi,

Thank you for reporting this issue. We will get to it as soon as possible.

Please let us know if you have any further issues.

**Comment by eithan1231** [comment]
I had a quick look into this issue and all appaers to be working as expected.

Can you please try upgrading to the latest version of `whois-light` to see if your issue is persistent?

**Comment by nenkoszai** [comment]
appears that the issue is the inconsistent format output that will result in not being able to resolve tld properly 
See the original issue https://github.com/eithan1231/whois-light/issues/4
I believe if you address the underlying format issue, we wont have these type of issues on many TLD domaisn

**Comment by eithan1231** [comment]
I reviewed issue #4 and it unfortunately is not an easy task to make a parser for all non-mainstream TLD whois servers. I had started implementing parsers for specific whois providers but there are hundreds of them, with no consistent format between them.

I will have to review it again in the future to see if there is an easier approach.

I might build a parsing base which will support mainstream TLDs, and accept PRs for non-mainstream TLDs.

**Comment by nenkoszai** [comment]
Yeah, i wouldnt mind participating in PRs for non-mainstream tld because we are seeing a significant increase of phishing domains using non mainstream tld. Im using your library to do check the domain registrar. Case in reference there was 10k phishing messages witht .fun being spammed in discord servers.
